.github/workflows/main.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           npm ci
           npm run format:check
 
-  get_docker_image_tags:
+  get_and_save_docker_image_tags:
```

Comment:
Just reading this and it's not 100% clear what this job is doing.   Maybe update this to `update_dev_docker_images`?

And then you could update the `name` below to: `Updates dev Docker image tags to latest` or something more succinct?